packages/databyss-ui/components/Menu/DropdownListItem.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { View, Text, Icon, BaseControl } from '@databyss-org/ui/primitives'
+import { pxUnits } from '@databyss-org/ui/theming/views'
+
+const DropdownListItem = ({ menuItem, onPress }) => (
+  <BaseControl
+    data-test-block-menu={menuItem.action}
+    onPress={onPress}
+    childViewProps={{ width: '100%' }}
+    px="small"
+    py="extraSmall"
+    hoverColor="background.1"
+    activeColor="background.1"
+  >
+    <View
+      flexDirection="row"
+      justifyContent="space-between"
+      width="100%"
+      alignItems="center"
+    >
+      <View flexDirection="row" alignItems="center">
+        {menuItem.textSymbol && (
+          <Text
+            variant="uiTextSmall"
+            width={pxUnits(20)}
+            textAlign="center"
+            mr="small"
+            color="text.2"
+          >
+            {menuItem.textSymbol}
+          </Text>
+        )}
+        {menuItem.icon && (
+          <Icon
+            sizeVariant="small"
+            width={pxUnits(20)}
```

Comment:
isn't the `width` specified by the size variant? maybe we need a new variant, like `menu`?

Reply:
Yeah, that was unnecessary to add since 20px is the size variant for small. I deleted it now :)